client/package.json

```diff
@@ -2,8 +2,13 @@
 	"name": "client",
 	"version": "1.0.0",
 	"dependencies": {
+		"@fortawesome/fontawesome-svg-core": "^1.2.17",
+		"@fortawesome/free-solid-svg-icons": "^5.8.1",
+		"@fortawesome/react-fontawesome": "^0.1.4",
```

Comment:
what are the needs for this?

Reply:
For the checkmark icons in the login page.

Reply:
I just used the cdn link to keep it simple elsewhere in the app so we wouldn't need to use both.